test/sql/copy/encryption/encrypted_to_unencrypted.test_slow

```diff
@@ -0,0 +1,55 @@
+# name: test/sql/copy/encryption/encrypted_to_unencrypted.test_slow
+# group: [encryption]
+
+require noforcestorage
+
+require skip_reload
+
+# workaround - alternative verify always forces the latest storage
+require no_alternative_verify
```

Comment:
I don't quite follow why this `require` is necessary here - I think this might be hiding some bugs (see comments above). Can we remove it?